source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3880,6 +3891,16 @@ Controls
    :range: binary, read-only
 
 
+.. mixxx:control:: [EffectRack1_EffectUnitN_EffectM],loaded_effect
+                   [EqualizerRack1_[ChannelI]_Effect1],loaded_effect
+                   [QuickEffectRack1_[ChannelI]_Effect1],loaded_effect
+
+   0-based index of the currently loaded effect preset, including the
+   empty/passthrough preset "\-\-\-".
+
+   :range: integer, 0 .. [:mixxx:coref:`num_effectsavailable<[Master],num_effectsavailable>` - 1]
```

Comment:
do we also have the fallback `-1` for just effects?

Reply:
For chain presets -1 means "loaded but modified".
Sure, effects can be modified, too, and the current state can be saved as new default, but IMO there's not much value in setting it to -1 as soon as the effect parameters are tweaked.

Reply:
yeah. I don't necessary see the value either, just asked out of curiosity.